### PR TITLE
Add a draft contributing.md

### PR DESCRIPTION
**Pull request by matozoid**
Anything to add or remove?

### CODE REVIEW

**Comment by ftomassetti** [comment]
You broke AppVeyor :)

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+Here is some information that's good to know when contributing to JavaParser.
+
+- if you're new and like to contribute something, check out the [easy issues](https://github.com/javaparser/javaparser/labels/Easy)
+- we work on JavaParser because we like to, not because it earns us money. Please remember that we try to run a professional project in our spare time, on a budget of zero.
+- if you know how to fix a problem, please fix it and open a pull request instead of opening an issue.
+- a pull request should include tests. You can either use BDD (JBehave) or JUnit.
+- there is some interesting information on the [wiki](https://github.com/javaparser/javaparser/wiki).
+- Be sure to check [the coding guidelines](https://github.com/javaparser/javaparser/wiki/Coding-Guidelines) which are easily used by installing the formatting rules as described there.
```
Comment (ftomassetti) [inline CONTRIBUTING.md]: this startes with a capital letter while the others don't

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+Here is some information that's good to know when contributing to JavaParser.
```
Comment (ftomassetti) [inline CONTRIBUTING.md]: should we use a colon here instead of the full stop?

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+
+- if you're new and like to contribute something, check out the [easy issues](https://github.com/javaparser/javaparser/labels/Easy)
+- we work on JavaParser because we like to, not because it earns us money. Please remember that we try to run a professional project in our spare time, on a budget of zero.
+- if you know how to fix a problem, please fix it and open a pull request instead of opening an issue.
+- a pull request should include tests. You can either use BDD (JBehave) or JUnit.
+- there is some interesting information on the [wiki](https://github.com/javaparser/javaparser/wiki).
```
Comment (ftomassetti) [inline CONTRIBUTING.md:9]: Should we say what kind of information is available on the wiki?

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,11 @@
+Here is some information that's good to know when contributing to JavaParser.
+
+- if you're new and like to contribute something, check out the [easy issues](https://github.com/javaparser/javaparser/labels/Easy)
+- we work on JavaParser because we like to, not because it earns us money. Please remember that we try to run a professional project in our spare time, on a budget of zero.
+- if you know how to fix a problem, please fix it and open a pull request instead of opening an issue.
+- a pull request should include tests. You can either use BDD (JBehave) or JUnit.
```
Comment (ftomassetti) [inline CONTRIBUTING.md]: Should we link to this page: https://github.com/javaparser/javaparser/wiki/Testing ?

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -0,0 +1,11 @@
...
+- there is some interesting information on the [wiki](https://github.com/javaparser/javaparser/wiki).
+- Be sure to check [the coding guidelines](https://github.com/javaparser/javaparser/wiki/Coding-Guidelines) which are easily used by installing the formatting rules as described there.
+- pull requests often stay open for at least a few days to give people a chance to review it.
+- a pull request is merged when all comments on it have been resolved.
```
Comment (ftomassetti) [inline CONTRIBUTING.md]: Should we specify that Travis and AppVeyor must be happy? It should be obvious but...

**Review by ftomassetti — APPROVED** [review]
Thanks for writing this!

File: CONTRIBUTING.md (hunk 3 above)
Comment (matozoid) [reply CONTRIBUTING.md:9]: We're mentioning BDD and the coding guidelines explicitly, do you think there's more info that's essential for committers?

File: CONTRIBUTING.md (hunk 2 above)
Comment (matozoid) [reply CONTRIBUTING.md]: Yes

File: CONTRIBUTING.md (hunk 1 above)
Comment (matozoid) [reply CONTRIBUTING.md]: Yes.

File: CONTRIBUTING.md (hunk 5 above)
Comment (matozoid) [reply CONTRIBUTING.md]: Yes.

File: CONTRIBUTING.md (hunk 4 above)
Comment (matozoid) [reply CONTRIBUTING.md]: Yes.

**Comment by matozoid** [comment]
AppVeyor is being a pain in the back: it has been broken for quite a while, but when I build on a Windows machine everything is fine...

**Pull request merged**